webapp-django/crashstats/signature/static/signature/js/signature_report.js

```diff
@@ -157,18 +157,16 @@ SignatureReport.init = function () {
         });
 
         // Show or hide filters.
-        searchSection.on('click', '.toggle-filters', function (e) {
+        $('.toggle-filters').click(function (e) {
             e.preventDefault();
-
-            var elt = $(this);
-            form.toggle();
-            elt.toggleClass('show');
-            if (elt.hasClass('show')) {
-                elt.html('Show');
-            }
-            else {
-                elt.html('Hide');
-            }
+            searchSection.slideToggle(350);
+
+            // Update the main toggle link.
+            var mainToggleLink = $('.display-toggle-filters');
+            mainToggleLink.toggleClass('show');
+            mainToggleLink.html(
+                (mainToggleLink.hasClass('show') ? 'Show' : 'Hide') + ' Search Filters'
```

Comment:
English in the js code isn't great. It means we're now spelling "Search Filters" in two places. 
(Generally, it's bad because it's hard to I18N static .js files but easy to do in the template rendering. However, that's not a concern in our case). 
I think you should change the HTML to be 
```html
<a href="#" class="display-toggle-filters toggle-filters show" 
  data-opposite="Hide Search Filters">Show Search Filters</a>
```
And then use:
```javascript
var opposite = mainToggleLink.data.opposite;
mainToggleLink.data.opposite = mainToggleLink.text();
mainToggleLink.text(opposite);
```